benchmarking/evaluations/metrics/average_timeslot_coverage.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
+
+from api.models.enums import ScenarioAttribute
+from api.models.team_set import TeamSet
+from benchmarking.evaluations.interfaces import TeamSetMetric
+
+
+class AverageTimeslotCoverage(TeamSetMetric):
+    """
+    Calculate the average mutual timeslots between students in a team and average over all teams.
+    """
+
+    def __init__(
+        self,
+        available_timeslots: List[int],
+        timeslot_attribute_value: int = ScenarioAttribute.TIMESLOT_AVAILABILITY.value,
```

Comment:
kind of a nit;, but it's super confusing to call this a value when it is, by definition, an id